### DISCUSSION TITLE
Newbie seeking help

### DISCUSSION THREAD

**Discussion by Cyberwebnerd** [organicmaps/organicmaps#9341] [Q&A]
I have been recommended github to help me learning about what resources are available to learn to hack.   Having looked through loads of interesting, such as ip-biter.

But, as someone who has learning difficulties, and can't read/understand code, I find them all to confusing of how to install where etc

**Comment by hb0nd** [comment]
this isn't a forum for general help, sorry.

**Reply by Cyberwebnerd** [reply]
Is there a way i can contact a repository creater for help/advice
